com/win32comext/authorization/demos/EditServiceSecurity.py

```diff
@@ -1,7 +1,7 @@
 """
 Implements a permissions editor for services.
 Service can be specified as plain name for local machine,
-or as a remote service of the form \\machinename\service
+or as a remote service of the form \\\\machinename\\service
```

Comment:
I think this is wrong too - it doesn't even pretend to be a string (ie, it's not quoted), it's a textual description of how Windows represents UNC paths, which is wrong with the slashes doubled.

Reply:
(same comment about docstring)
`\s` is an invalid escape sequence in a string (this is a multiline string)
Before:
```python
>>> from win32comext.authorization.demos import EditServiceSecurity
>>> print(EditServiceSecurity.__doc__)

Implements a permissions editor for services.
Service can be specified as plain name for local machine,
or as a remote service of the form \machinename\service
```
![image](https://github.com/mhammond/pywin32/assets/1350584/4095574e-1976-40ed-82ec-89a77a7cd9d1)


After:
```python
>>> from win32comext.authorization.demos import EditServiceSecurity
>>> print(EditServiceSecurity.__doc__)

Implements a permissions editor for services.
Service can be specified as plain name for local machine,
or as a remote service of the form \\machinename\service
```